cwms-data-api/src/test/resources/cwms/cda/data/dto/watersupply/water_supply_accounting.json

```diff
@@ -0,0 +1,95 @@
+{
+  "contract-name": "Test Contract",
+  "water-user": {
+    "entity-name": "Test Entity",
+    "project-id": {
+      "office-id": "SPK",
+      "name": "Test Location"
+    },
+    "water-right": "Test Water Right"
+  },
+  "pump-in-accounting": {
+    "Test Pump": {
+      "pump-location": {
+        "office-id": "SPK",
+        "name": "Test Location-Test Pump"
+      },
+      "pump-transfers": {
+        "2286-11-20T21:17:28Z": {
+          "transfer-type-display": "Test Transfer Type",
+          "flow": 1,
+          "transfer-date": 10000012648000,
+          "comment": "Test Comment"
+        },
+        "2286-11-20T21:17:29Z": {
```

Comment:
what is the difference between this date and the value in "transfer-date"

Reply:
The dates are the same - it's duplicate data. It should be removed in the next iteration of the DTO based on the timeseries-style JSON format currently being tested